boards.txt

```diff
@@ -711,6 +711,70 @@ adafruit_feather_m4.menu.debug.off=Off
 adafruit_feather_m4.menu.debug.on=On
 adafruit_feather_m4.menu.debug.on.build.flags.debug=-g
 
+# Adafruit Feather M4 CAN (SAME51)
+# ------------------------------
+adafruit_feather_m4_can.name=Adafruit Feather M4 CAN (SAME51)
+adafruit_feather_m4_can.vid.0=0x239A
+adafruit_feather_m4_can.pid.0=0x80CD
+adafruit_feather_m4_can.vid.1=0x239A
+adafruit_feather_m4_can.pid.1=0x00CD
+adafruit_feather_m4_can.upload.tool=bossac18
+adafruit_feather_m4_can.upload.protocol=sam-ba
+adafruit_feather_m4_can.upload.maximum_size=507904
+adafruit_feather_m4_can.upload.offset=0x4000
+adafruit_feather_m4_can.upload.use_1200bps_touch=true
+adafruit_feather_m4_can.upload.wait_for_upload_port=true
+adafruit_feather_m4_can.upload.native_usb=true
+adafruit_feather_m4_can.build.mcu=cortex-m4
+#adafruit_feather_m4_can.build.f_cpu=120000000L
+adafruit_feather_m4_can.build.usb_product="Adafruit Feather M4 CAN"
+adafruit_feather_m4_can.build.usb_manufacturer="Adafruit LLC"
+adafruit_feather_m4_can.build.board=FEATHER_M4
+adafruit_feather_m4_can.build.core=arduino
+adafruit_feather_m4_can.build.extra_flags=-D__SAME51J19A__ -DADAFRUIT_FEATHER_M4_EXPRESS -DADAFRUIT_FEATHER_M4_CAN -D__SAMD51__ {build.usb_flags} -D__FPU_PRESENT -DARM_MATH_CM4 -mfloat-abi=hard -mfpu=fpv4-sp-d16
```

Comment:
should the `-DADAFRUIT_FEATHER_M4_EXPRESS` be removed

Reply:
no, because its is fully back compatble

Reply:
got it

Reply:
@ladyada suggested to have both, if I understood correctly